EnglishFilter/sections/antiadblock.txt

```diff
@@ -648,6 +648,8 @@ highload.to#%#//scriptlet('prevent-fetch', 'popcash.net')
 gmailnator.com#%#//scriptlet("abort-current-inline-script", "onload", "adblock")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/93416
 cryptofans.news#%#//scriptlet('prevent-setTimeout', 'swal')
+! https://github.com/AdguardTeam/AdguardFilters/issues/95426
+animesaria.com#%#//scriptlet('prevent-setTimeout', 'testAd')
```

Comment:
Hello @scripthunter7, it's enough and better to use this rule `animesaria.com#@#.adsbygoogle`, because f.e. AdGuard for iOS doesn't support scriptlets.

Reply:
Hello @Sergey-Lyapin! It is "safe" / good idea to add this class to the exceptions?

Reply:
@scripthunter7 
Yes, because the site uses this class for adblock detection only.

Reply:
I changed the rule by myself I hope you don't mind.